scipy/optimize/_linprog_simplex.py

```diff
@@ -431,42 +432,39 @@ def _linprog_simplex(c, c0, A, b, maxiter=1000, disp=False, callback=None,
     b : 1D array
         1D array of values representing the right hand side of each equality
         constraint (row) in ``A``.
-    callback : callable, optional (simplex only)
+    callback : callable, optional
         If a callback function is provided, it will be called within each
-        iteration of the simplex algorithm. The callback must require a
+        iteration of the algorithm. The callback function must accept a single
         `scipy.optimize.OptimizeResult` consisting of the following fields:
 
             x : 1D array
-                The independent variable vector which optimizes the linear
-                programming problem.
+                Current solution vector
             fun : float
-                Value of the objective function.
+                Current value of the objective function
             success : bool
-                True if the algorithm succeeded in finding an optimal solution.
+                True when an algorithm has completed successfully.
             slack : 1D array
                 The values of the slack variables. Each slack variable
                 corresponds to an inequality constraint. If the slack is zero,
                 the corresponding constraint is active.
             con : 1D array
-                The (nominally zero) residuals of the equality constraints, that
-                is, ``b - A_eq @ x``
+                The (nominally zero) residuals of the equality constraints,
+                that is, ``b - A_eq @ x``
             phase : int
-                The phase of the optimization being executed. In phase 1 a basic
-                feasible solution is sought and the T has an additional row
-                representing an alternate objective function.
+                The phase of the algorithm being executed.
             status : int
-                An integer representing the exit status of the optimization::
+                An integer representing the status of the optimization::
 
-                     0 : Optimization terminated successfully
+                     0 : Algorithm proceeding nominally
```

Comment:
Should this be "Algorithm proceeding *normally*"

Reply:
In certain contexts, 'nominal' means something like "within acceptable bounds"/normal.  I can change this, though, if this is not a familiar use of the word.

Reply:
Given the context either should be OK, at this point I don't think there will be any real benefit to changing it.